proto/viam/app/build/v1/build.proto

```diff
@@ -83,3 +99,79 @@ message ListJobsResponse {
   // jobs is ordered by (build start time, alphabetical platform).
   repeated JobInfo jobs = 1;
 }
+
+message RepoLink {
+  // viam internal ID which maps to an external oauth app
+  string oauth_app_link_id = 1;
+  // OrgID of the module
+  string org_id = 2;
+  // public namespace of the module
+  optional string namespace = 3;
```

Comment:
[probably worth discussing] Why is associating the namespace necessary for this message (same for AppLink)?

I worry that this will mean that will add a lot more logic & edge cases for module transfers from org to org.

Reply:
this is populated in the downstream direction so that CLI output is user-readable; it's not stored in the DB and it's not used in the request. I'll edit the docstring to say this

Reply:
SGTM